src/main/java/org/fundacionjala/core/utils/PropertiesReader.java

```
@@ -0,0 +1,59 @@
+package org.fundacionjala.core.utils;
+
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+public class PropertiesReader {
+
+    private static final Logger LOGGER = LogManager.getLogger(PropertiesReader.class);
+    private Properties property;
+    private FileReader reader;
+
+    /**
+     * Constructor.
+     * @param pathFile
+     */
+    public PropertiesReader(final String pathFile) {
+        try {
+            reader = new FileReader(pathFile);
+            property = new Properties();
+            property.load(reader);
+        } catch (FileNotFoundException e) {
+            LOGGER.error(e.getMessage());
+        } catch (IOException e) {
+            LOGGER.error(e.getMessage());
+        } finally {
```

I think that it should also throw an exception